refs/schemas/priv/expected-behavior.md

```diff
@@ -0,0 +1,655 @@
+```
+
+The Data Subject realizes that they are receiving too much advertising material and they formulate a Privacy Request with one `REVOKE-CONSENT` demand targeting particular consent that they gave.
+
+```
+Eligible Privacy Scope:
+CONTACT.EMAIL x * x MARKETING (LEGITIMATE-INTEREST)
+CONTACT.EMAIL x * x SERVICES (CONTRACT)
+CONTACT.ADDRESS x * x SERVICES (CONTRACT)
+```
+
+The Data Subject realizes that they are still receiving marketing e-mail and they formulate a Privacy Request with one `OBJECT` demand with `CONTACT.EMAIL` being defined as its Privacy Scope. Yet, this Demand only affects the [Privacy Scope Triples](#privacy-scope-triples) held under `LEGITIMATE-INTEREST`. The System can still, continue to keep the `CONTACT.EMAIL` and `CONTACT.ADDRESS` for the same of the ongoing contractual relationship with the user.
+
+```
+Eligible Privacy Scope:
+CONTACT.EMAIL x * x SERVICES (CONTRACT)
+CONTACT.ADDRESS x * x SERVICES (CONTRACT)
+```
+However, the remaining purpose (`SERVICES`) now only allows the System to send e-mail related to the services being provided, and no longer supports sending marketing e-mail to this Data Subject.
+
+### Calculating Permissions
+
+Since the [Privacy Compiler](../high-level-architecture#data-rights-compiler) maintains the Eligible Privacy Scope at any time, it can (given a Data Capture ID, or a Data Capture Fragment ID) answer whether at that time, a particular data processing operation is permitted.
+
+For example, the System can inquire whether a particular data fragment, corresponding to `CONTACT.EMAIL` data category, can be used to invoice the user (processing = `USING`; purpose = `SERVICES`), or whether it can be shared with some other partner system for user profiling (processing = `SHARING`; purpose = `AUTOMATED-INFERENCE`).
+
+
+## Resolving requests
+
+The [Privacy Compiler](../high-level-architecture#data-rights-compiler) evaluates Privacy Request Demands and issues recommendations to the System. The Systems can then be configured to respond to Privacy Requests automatically, blindly following those recommendations (whenever possible), or to await human input just in case. Cf. [different automation scenarios](./scenarios.md#automation)
+
+Privacy Requests MAY need to be processed in the context of different [authentication scenarios](./scenarios.md#authentication) and different [response scenarios](./scenarios.md#response).
+
+Here is how the Privacy Request Response recommendations are calculated:
+
+When no Data Subject ID is provided in the Privacy Request:
+- `TRANSPARENCY` Demands: recommend status = `GRANTED`. Provide requested information: Data Categories, Processing Categories, Purposes, and Legal Bases from Intended Privacy Scope, as well as other information from general settings as defined under [Implications for Systems](./RFC-PRIV.md#design-implications-for-systems-implementing-PRIV) and under [Configuration and Prerequisites](#configuration-and-prerequisites).
```

Comment:
According to [definitions](https://github.com/blindnet-io/product-management/blob/devkit-schemas/refs/schemas/priv/dictionary/actions/en.actions.json) some of the `TRANSPARENCY` demands require Data Subject to be known?

Reply:
The Definitions are NOT NORMATIVE.

According to [GDPR.13 & GDPR.14](https://github.com/blindnet-io/product-management/blob/devkit-schemas/refs/schemas/priv/examples.md#articles-13-and-14) Systems MUST respond to some TRANSPARENCY requests in one way during and beofre data processing (i.e. when the user is still unknown) and  in a different way, according to [GDPR.15](https://github.com/blindnet-io/product-management/blob/devkit-schemas/refs/schemas/priv/examples.md#article-151) when user is known. So either way, the status is `GRANTED`